gam/gam.py

```diff
@@ -54,7 +55,7 @@ class GAM:
     def __init__(
         self,
         k=2,
-        attributions_path="local_attributions.csv",
+        attributions_path=None,
         attributions=None,
```

Comment:
@hleerighter8 We still want the ability to pass in a path to GAM, so `attributions_path` is fine. I'll need to update the docstring for `attributions` though, because we also do want the ability to pass in a dataframe or array, so I don't really want to change that to attributions_df since it could be either one of them.